Makefile

```diff
@@ -23,3 +24,14 @@ test:
 .PHONY: lint
 lint:
 	docker run --rm -v $$(pwd):/app -w /app golangci/golangci-lint:v1.46.2 golangci-lint run -v
+
+# Builds multiple architectures at once.
+# Requires docker buildx and QEMU to be configured.
+# Because of how docker buildx works, in that it _must_ push when it builds, push is the default of this task.
+.PHONY: buildx
+buildx:
+	@if [ -z "$$VERSION" ]; then \
+		echo "VERSION is required"; \
+		exit 1; \
+	fi
+	docker buildx build --platform linux/amd64,linux/arm64 --push .
```

Comment:
Are these the only two platforms we want to support initially?

Reply:
these 2 platforms are the easiest to support out of the gate, I figured more could be added later

Reply:
Also, oops, just realized that I forgot the tags, I think we need to add this to the command: `-t $(IMAGE_REPO):$$VERSION`